src/tools/schema.js

```diff
@@ -5,6 +5,28 @@ const cleaner = require('./cleaner');
 const dataTools = require('./data');
 const zapierSchema = require('zapier-platform-schema');
 
+// method is {display: {}, operation: {}}
+// methodMap is {key: {display: {}, operation: {}}}
+const walkMethods = (resource, methodOrMethodMap) => {
+  methodOrMethodMap = methodOrMethodMap || {};
+
+  if (methodOrMethodMap.operation) {
+    const method = dataTools.deepCopy(methodOrMethodMap);
+    method.key = resource.key;
+    return [method];
+  }
+
+  if (Object.keys(methodOrMethodMap).length) {
```

Comment:
Would probably use `_.isEmpty(methodorMethodMap)` instead

Reply:
and actually, don't even really need this check here, since `Object.keys({}).map(...)` returns `[]` anyway. Though it might be worth keeping from a clarity standpoint.

Reply:
Oh, good points!